pkg/kubectl/cmd/util/printing.go

```diff
@@ -28,12 +29,12 @@ import (
 
 // AddPrinterFlags adds printing related flags to a command (e.g. output format, no headers, template path)
 func AddPrinterFlags(cmd *cobra.Command) {
-	cmd.Flags().StringP("output", "o", "", "Output format. One of: json|yaml|template|templatefile|wide|jsonpath.")
+	cmd.Flags().StringP("output", "o", "", "Output format. One of: json|yaml|wide|gotemplate=...|gotemplate-file=...|jsonpath=...|jsonpath-file=... See golang template [http://golang.org/pkg/text/template/#pkg-overview] and jsonpath template [http://releases.k8s.io/HEAD/docs/user-guide/jsonpath.md].")
```

Comment:
Jenkins failed in post-cluster creation setup with this error:
```
F0826 08:30:48.611573    4970 command.go:476] template format specified but no template given
```

The problem is that there are 2 separate flags which you have accidentally mushed into one. There is the output flag `-o gotemplate` and the the template path|string flag `-t`. You have to specify both.

On that note, remove the trailing `=` from `gotemplate|gotemplate-file|jsonpath|jsonpath-file`.